src/chat.rs

```diff
@@ -1611,6 +1612,39 @@ pub async fn send_text_msg(
     send_msg(context, chat_id, &mut msg).await
 }
 
+pub async fn send_videochat_invitation(context: &Context, chat_id: ChatId) -> Result<MsgId, Error> {
+    ensure!(
+        !chat_id.is_special(),
+        "videochat invitation cannot be sent to special chat: {}",
+        chat_id
+    );
+
+    let url = if let Some(basic_webrtc_instance) =
+        context.get_config(Config::BasicWebrtcInstance).await
+    {
+        basic_webrtc_instance
+    } else {
+        bail!("basic_webrtc_instance not set");
+    };
+
+    let room = dc_create_id();
+
+    let url = if url.contains("$ROOM") {
+        url.replace("$ROOM", &room)
+    } else {
+        format!("{}{}", url, room)
+    };
```

Comment:
So the config var needs to be sth. like `https://basic-web-rtc.server/p2p/#roomname=$ROOM&camon=true`?

Reply:
or `https://basic-web-rtc.server/p2p/#roomname=` - if $ROOM is left out, it is appended.